zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -61,6 +62,30 @@ class MeasurementType(enum.IntFlag):
         POWER_QUALITY_MEASUREMENT = 256
 
     REPORT_CONFIG = (
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_voltage_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_voltage_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_current_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_current_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_power_multiplier.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
+        AttrReportConfig(
+            attr=ElectricalMeasurement.AttributeDefs.ac_power_divisor.name,
+            config=REPORT_CONFIG_IMMEDIATE,
+        ),
```

Comment:
I put these at the very beginning just in case the device's attribute reports are sent in the order of the reporting config. If this is the case, you may need to re-join the device to get this to work.